src/main/java/com/minecolonies/coremod/permissions/ColonyPermissionEventHandler.java

```diff
@@ -373,7 +374,8 @@ public void on(final ArrowLooseEvent event)
     public void on(final AttackEntityEvent event)
     {
         final EntityPlayer playerIn = event.getEntityPlayer();
-        if (Configurations.enableColonyProtection && colony.isCoordInColony(playerIn.getEntityWorld(), playerIn.getPosition()))
+        if (Configurations.enableColonyProtection && colony.isCoordInColony(playerIn.getEntityWorld(), playerIn.getPosition())
+                && !(event.getEntity() instanceof EntityMob))
```

Comment:
Move this check to be second, so we don't have to do the isCoordInColonyCheck for mobs